curvefs/src/mds/fs_manager.cpp

```diff
@@ -793,5 +870,41 @@ void FsManager::CommitTx(const CommitTxRequest* request,
     dlock_->UnLock(fsName, uuid);
 }
 
+void FsManager::UpdateClientAliveTime(const Mountpoint& mountpoint,
+    const std::string& fsName, bool remove) {
+    VLOG(1) << "UpdateClientAliveTime fsName = " << fsName
+            << ", mp = " << mountpoint.DebugString() << ", remove = " << remove;
+    WriteLockGuard wlock(recorderMutex_);
+    std::string mountpath;
+    MountPoint2Str(mountpoint, &mountpath);
```

Comment:
This sentence does not need to be in the lock

Reply:
fixed